template/package.json

```diff
@@ -93,7 +96,7 @@
     "cacheDirectory": ".jest/cache"
   },
   "lint-staged": {
-    "**/*.{ts, tsx}": [
+    "**/*.{ts,tsx}": [
```

Comment:
if you made a commit with only `tsx` file changes, it was not triggering lint-staged